src/backend/dx11/src/factory.rs

```diff
@@ -599,7 +599,7 @@ impl core::Factory<R> for Factory {
     }
 
     fn create_texture_raw(&mut self, desc: texture::Info, hint: Option<core::format::ChannelType>,
-                          data_opt: Option<&[&[u8]]>) -> Result<h::RawTexture<R>, texture::CreationError> {
+                          data_opt: Option<(&[&[u8]], texture::Mipmap)>) -> Result<h::RawTexture<R>, texture::CreationError> {
```

Comment:
clearly, D3D11 needs more changes to get this working
at least you can return an error if it's `Mipmap::Generated`

Reply:
This will narrow the code portability, so the transfer GL->d3d11 will require code patch, while the mode is actually supported, just unparameterized.

Reply:
I don't see how this mode is "actually supported" on D3D11. We construct `D3D11_SUBRESOURCE_DATA` per data slice. Having less of those just out of nowhere would cause the confusion in the driver.